tests/sync/app.cpp

```diff
@@ -299,7 +716,8 @@ struct ErrorCheckingTransport : public GenericNetworkTransport {
     Response m_response;
 };
 
-
+//FIXME: - Broken tests
```

Comment:
If possible please enable these tests. But if the errors are not related to your work, we can fix them in a separate PR. (not blocking)